kolibri/plugins/learn/assets/src/views/LearnSideNavEntry.vue

```diff
@@ -16,63 +17,25 @@
   import navComponents from 'kolibri.utils.navComponents';
   import urls from 'kolibri.urls';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import { PageNames as LearnPageNames } from '../constants';
-  import { generateNavRoute } from '../../../../../core/assets/src/utils/generateNavRoutes';
   import baseRoutes from '../routes/baseRoutes';
   import commonLearnStrings from './commonLearnStrings';
 
-  const url = urls['kolibri:kolibri.plugins.learn:learn']();
   const component = {
     name: 'LearnSideNavEntry',
     mixins: [commonLearnStrings, commonCoreStrings],
     components: {
       CoreMenuOption,
     },
     computed: {
-      learnRoutes() {
```

Comment:
I think you've gone too far in just using the static baseRoutes. You need to be able to reference strings here in order to get the labels in a sufficiently flexible way, and there's also no specification of 'show on bottom bar' in the baseRoutes you have defined.